sass/components/_dots.scss

```diff
@@ -0,0 +1,26 @@
+.dots {
+    .dot {
+        height: 12px;
+        width: 12px;
+        background-color: $col-background;
+        border-radius: 50%;
+        display: inline-block;
+        cursor: pointer;
+
+        &--checked {
+            background-color: $col-text-600;
+        }
+
+        &:nth-child(2) {
+            margin: 0 1.2rem 0 1.2rem;
+        }
```

Comment:
to jest rozwiązanie konkretnie na 3 kropki,
powinno być uniwersalne, tak jakby miała być dowolna ilość kropek